tests/integration/conftest.py

```diff
@@ -127,6 +128,7 @@ def config():
                 Keys.ROW_PROBABILITY: ROW_PROBABILITY,
             }
             for noise_type in dataset.row_noise_types
+            if noise_type != NOISE_TYPES.duplicate_with_guardian
```

Comment:
Why is this here?

Reply:
THis fixture is just increasing the row noise level from the default 0 to 5%. The 3 groups for guardian duplication already default to that level. The other option would be to iterate through the 3 group keys and set it to 5% (which would do nothing but maybe more clear).